agent/app/agent_capability.go

```diff
@@ -51,7 +51,8 @@ const (
 	capabilitySecretLogDriverSSM                           = "secrets.ssm.bootstrap.log-driver"
 	capabilitySecretLogDriverASM                           = "secrets.asm.bootstrap.log-driver"
 	capabiltyPIDAndIPCNamespaceSharing                     = "pid-ipc-namespace-sharing"
-	capabilityNvidiaDriverVersionInfix                     = "nvidia-driver-version."
+	capabilityNvidiaDriverVersion                          = "nvidia-driver-version"
```

Comment:
Hi @risdenk ,

Thank you for making the contribution. 

Can you name the new attribute 'gpu-driver-version' instead of 'nvidia-driver-version' so that there is no overlap between the two?

Reply:
There is no overlap in the keys from a practical perspective. The existing keyname is:

`nvidia-driver-version.470.57.02`

This is the full key name. The new keyname is just `nvidia-driver-version`.

I'm not sure that generic `gpu-driver-version` makes sense? I can make the change, but I don't understand the overlap concern.

Reply:
Hi @risdenk, in regards to the overlap concern - our team discussed and we think that users who want to filter with `nvidia-driver-version*` should not get the new key name but the existing key name. So we think it would be best to have the new key name renamed to `gpu-driver-version` instead to avoid confusion between the existing and new key names. 

Furthermore, in the future we may also consider adding GPU types other than NVIDIA. Perhaps we might look into having something like `gpu-driver-type` set to `nvidia` and `gpu-driver-version` set to `470.10.*` for example.

Reply:
@danehlim that's fine but that is a drastically different change than what is proposed here. If AWS wants to make that drastic change then go for it.

Reply:
@risdenk Just to clarify, we are not requesting support of multiple GPU manufacturers to be added to this pull request. We are just requesting a change in naming to avoid the concerns in overlap mentioned previously and to not close the door on supporting multiple GPU manufacturers in the future. If you have any concerns with this, please feel free to let us know.